docs/getting-started/development.md

```diff
@@ -249,24 +127,24 @@ $ vagrant rsync selenium-node-chrome
 $ vagrant rsync selenium-node-firefox
 ```
 
-Run the tests with:
+Run the tests with e.g.:
 
 ```
 $ vagrant ssh app-debug-1 --command 'current/vendor/bin/behat --config data/behat/behatrun/behat/behat.yml --profile chrome'
 ```
 
 ### PHPUnit
 
-With the relevant configuration options present in your Moodle `config.php`, run the following to enable PHPUnit:
+Review the PHPUnit-related options in your Moodle `config.php` (see the recommended configuration for advice) and run the following command to bootstrap the testing environment:
 
 ```
-$ vagrant ssh app-debug-1 --command 'sudo -u moodle php ~moodle/htdocs/admin/tool/phpunit/cli/init.php'
+$ vagrant ssh app-debug-1 --command 'php ~/releases/vagrant/admin/tool/phpunit/cli/init.php'
 ```
 
-You may then run tests as follows:
+Run the tests with e.g.:
 
 ```
-$ vagrant ssh app-debug-1 --config 'sudo -u moodle php ~moodle/htdocs/vendor/bin/phpunit'
+$ vagrant ssh app-debug-1 --command '~/releases/vagrant/vendor/bin/phpunit -c ~/releases/vagrant/ --group=core_group'
```

Comment:
Please remove the `--group`. Again, since f77add6508de746fca62a4c393daf62e058020f7 you should be able to use `~/current` rather than `~/releases/vagrant`.